summer19/faq.html

```diff
@@ -5,38 +5,50 @@
-        <p class="">A: Each project is around two-month long.</p>
+        <p class="">A: You will be working on the problems for a prospective duration of 2 months. You will 
+          be paired with others interested in working on the project, plus we'll be there to help you out of you get stuck :)</p>
+        
         <p class="que">Q: What is the eligibility criteria?</p>
-        <p class="">A: There's no eligibility criteria . Everyone is welcome to contribute . However please fill out <a href = "https://docs.google.com/forms/d/1PrGlfqy6kB2Ap_2j2vu91U0aLsybBI1cZB8umqlY5Ok/edit" style="color:red">this</a> google form if you ish to participate.
+        <p class="">A: You're welcome to work with us even if you're in school! We have working professionals as well as 
+          awesome school students collaborating on many projects. [<a href = "https://docs.google.com/forms/d/1PrGlfqy6kB2Ap_2j2vu91U0aLsybBI1cZB8umqlY5Ok/edit" style="color:red" target="_blank">Hop on!</a>]</p>
+        
         <p class="que">Q: How much time do I need to dedicate to the project?</p>
         <p class="">A: There's no hard and fast rule for committing a bare minimum number of hours . <br>You can fix up the number of hours per week according to what suits you .</p>
         <p class="que">Q: Can I be a mentor for this programme?</p>
-        <p class="">A: Definitely , just select the role as mentor in the google doc and we will set you up.</p>
+        <p class="">A: Definitely , just select the role accordingly in the google doc and we will set you up.</p>
         <p class="que">Q: Is there a stipend for the programme?</p>
-        <p class="">A: No, this is just an initiative for promoting open-source cultue along with doing a solid project. There's no stipend since this is just to hone your skills.</p>
-        <p class="que">Q: what are the Starting and ending dates of the program?</p>
+        <p class="">A: No, this is just an initiative for promoting open-source culture along with doing a solid project. There's no stipend since this is just to hone your skills.</p>
+        <p class="que">Q: What are the starting and ending dates of the program?</p>
         <p class="">A: We will announce it very soon , just stay tuned.</p>
-        <p class="que">Q: How many mentors will be there for eaach project?</p>
+        <p class="que">Q: How many mentors will be there for each project?</p>
         <p class="">A: This will depend on the number of people who apply for the role of mentor . We will let you know when everything is finalized .</p>
+        <p class="que">Q: Will the project be done remotely?</p>
+        <p class="">A: Yes , this is an open source project and will be accomplished remotely. </p>
+        <p class="que">Q: Will anybody who has applied for the role of mentor be allowed to mentor?</p>
+        <p class="">A: No , once the registration for is closed , we will see and sort out people who will finally be mentoring. </p>
```

Comment:
There is nothing wrong with the above answer but I feel, the tone is a little bit harsh, replace it with something like, `Once the registrations are closed, mentors will be shortlisted by us according to the projects selected.` and replace the question with something like, `How will be the projects and mentors selected?`